ci-operator/config/kubevirt-ui/kubevirt-plugin/kubevirt-ui-kubevirt-plugin-main.yaml

```diff
@@ -2,7 +2,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: console-plugin-test-cypress
+    tag: tectonic-console-builder-28
```

Comment:
Here this tag should be `tectonic-console-builder-v28`. This is what is mentioned in the release.yaml.

Reply:
thanks man !! You've just saved my day :D